### DISCUSSION TITLE
Use of _bcg

### DISCUSSION THREAD

**Discussion by Pwday** [cortexrd/Knack-Toolkit-Library#163] [General]
I have other Keywords working but I cannot get this one to work. I place the  _bcg=,,h in the descritpion of the View (Details) I want to generate the QR code (First field) but nothings happens. Is there a role that is associated with this or am I placing in wrong area.

Thanks

Pete

**Comment by cortexrd** [comment]
Peter,

The bug has been fixed in **KTL v0.19.5**.

BTW, please use the Issues page when reporting problems, even if you're not sure it's a bug or something you don't fully understand.

Cheers,
Norm
